ctapipe/image/ellipsoid.py

```diff
@@ -0,0 +1,443 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: UTF-8 -*-
+"""
+Shower image parametrization based on image fitting.
+"""
+
+from enum import Enum
+
+import astropy.units as u
+import numpy as np
+from astropy.coordinates import Angle
+from iminuit import Minuit
+
+from ctapipe.image.cleaning import dilate
+from ctapipe.image.hillas import hillas_parameters
+from ctapipe.image.pixel_likelihood import (
+    mean_poisson_likelihood_gaussian,
+    neg_log_likelihood_approx,
+)
+from ctapipe.image.toymodel import SkewedGaussian, SkewedGaussianLaplace
+
+from ..containers import ImageFitParametersContainer
+
+PED_TABLE = {
+    "LSTCam": 2.8,
+    "NectarCam": 2.3,
+    "FlashCam": 2.3,
+    "SST-Camera": 0.5,
+    "CHEC": 0.5,
+    "DUMMY": 0,
+    "testcam": 0,
+}
+SPE_WIDTH = 0.5
```

Comment:
I guess this is also not the correct value, and you should get it from the appropriate Container.

Reply:
I also think this is not available. At least I could not find the SPE width